db/migrate/201703030109520_add_order_to_questionnaire_sections.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrderToQuestionnaireSections < ActiveRecord::Migration[5.0]
```

Comment:
We can just add this column into the previous migration, no need to create a new one :)